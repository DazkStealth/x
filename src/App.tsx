import React from 'react';
import Providers from './providers';
import Home from './pages/Home';

const App = () => {
  return (
    <Providers>
      {/* TODO: add router /> */}
      <Home />
    </Providers>
  );
}

export default App;
