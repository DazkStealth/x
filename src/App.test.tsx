import React from 'react';
import renderer from 'react-test-renderer';

// components
import App from './App';

describe('<App />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

