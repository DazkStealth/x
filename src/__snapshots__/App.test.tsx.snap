// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
.c0 {
  margin-top: calc(50% - 150px);
  display: flex;
  justify-content: center;
}

.c1 {
  font-size: 50px;
  font-weight: 700;
  width: 150px;
  height: 150px;
  line-height: 140px;
  border-radius: 50%;
  background: #fff;
  color: #997cfa;
  text-align: center;
  animation: fEdePq 5s ease-in-out infinite;
}

.c1:before {
  content: 'Px';
}

<div
  className="c0"
>
  <div
    className="c1"
  />
</div>
`;
